### ISSUE TITLE
Semantic issues

### ISSUE THREAD

**Issue by Armand-Lluka** [fac-14/team4week2#21]
Perhaps a stretch goal to fix, but the site doesn't have semantic markers. Maybe find a way of making list items readable on screen readers?
